test/rfxcom.spec.js

```diff
@@ -466,14 +606,14 @@ describe("RfxCom", function() {
             describe("device type identification", function() {
                 it("should identify HomeEasy EU devices", function(done) {
                     device.on("lighting2", function(evt) {
-                        expect(evt.subtype).toBe(rfxcom.lighting2.HOMEEASY_EU);
+                        expect(evt.subtype).toBe(rfxcom.lighting2[1]);
```

Comment:
I agree that subtype in lighting1Handler shouldn't be a string...and that's something I'm happy to break backwards compatibility for.